File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    let l=nums.length
    let result=[]
    for(let i=0;i<l-2;i++){
        for(let j=i+1;j<l-1;j++){
            for(let k=j+1;k<l;k++){
                result.push(nums[i]+nums[j]+nums[k])
            }
        }
    }
    let final=result.filter(item=>{
        let prime=true
        for(let i=2;i<item;i++){
            if(item%i===0)
                prime=false
        }
        if(prime===true)
            return item
    })
    
    return final.length;
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
from itertools import combinations
def solution(nums):
    l=list(combinations(nums,3))
    result=[]
    for item in l:
        result.append(sum(item))
    m=max(result)
    k=set(range(2,m+1))
    for i in range(2,m+1):
        if i in k:
            k=k-set(range(i*i,m+1,i))
    count=0
    for item in result:
        if item in k:
            count+=1
    return count
        